Mathlib/Topology/Category/TopCat/Basic.lean

```diff
@@ -134,8 +134,8 @@ set_option linter.uppercaseLean3 false in
 def homeoOfIso {X Y : TopCat.{u}} (f : X ≅ Y) : X ≃ₜ Y where
   toFun := f.hom
   invFun := f.inv
-  left_inv x := by simp
-  right_inv x := by simp
+  left_inv x := by simp [←comp_apply, id_apply]
+  right_inv x := by simp [←comp_apply, id_apply]
```

Comment:
We should instead have

```
attribute [elementwise (attr := simp)] Iso.hom_inv_id Iso.inv_hom_id
```
somewhere central. (I guess `CategoryTheory.ConcreteCategory.Elementwise`.)

Perhaps other places where you had to rewrite by `←comp_apply` could also benefit from more `elementwise` attributes.

Reply:
Or adding `attribute [elementwise (attr := simp)] Iso.hom_inv_id Iso.inv_hom_id` will make 
```
  left_inv x := by simp
  right_inv x := by simp
```
work